### DISCUSSION TITLE
Opsegenie_tags by default with config parameter

### DISCUSSION THREAD

**Discussion by rschirin** [jertel/elastalert2#540] [Ideas]
Hey there,
currently, the opsgenie_tags is set by default with the values "ElastAlert" and "{opsgenie_rule} in the file _elastalert/alerters/opsgenie.py_ and the code `self.tags = self.rule.get('opsgenie_tags', []) + ['ElastAlert', self.rule['name']]`

Are these tags really necessary? Maybe is it possible to configure them using a parameter into the opsgenie.py file?

**Comment by jertel** [comment]
I don't know if they are 'necessary' but they likely were helpful to the person who put them in the code several years ago. If you want to make them optional, but default to including them (so that it doesn't break existing users), feel free to submit a PR. Review the [Contribution Guidelines](https://github.com/jertel/elastalert2/blob/b075b47c655767c86ab2c40904960ddb896c8883/CONTRIBUTING.md) before submitting a PR.
